pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -158,8 +170,35 @@ var (
 			secret.Name = strings.ReplaceAll(secret.Name, hash, "<hash>")
 		},
 	}
+	RewriteRefreshPeriod = &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			humanizeRefreshPeriodFromMetadata(secret.Annotations)
+		},
+		rewriteConfigMapFn: func(configMap *corev1.ConfigMap) {
+			humanizeRefreshPeriodFromMetadata(configMap.Annotations)
+		},
+	}
 )
 
+func humanizeRefreshPeriodFromMetadata(annotations map[string]string) {
+	if annotations == nil {
+		annotations = map[string]string{}
```

Comment:
let's return here, in go a `nil` map is equivalent to an empty map from which reading is possible.

Reply:
ah, in that case we don't need this check at all - `annotations[certrotation.CertificateRefreshPeriodAnnotation]` will guard us. Added a unit test